site/content/en/docs/examples/tas.md

```diff
@@ -10,59 +10,37 @@ running workloads efficiently. The bandwidth between the Pods depends on the pla
 
 This example will cover how to deploy a vLLM multi-host workload using TAS.
 
-## Install the Kueue Controller
+## Define topology levels
+In a yaml file, define the different levels of the topology, the type of resource you will schedule on, and the name of the ClusterQueue that will be created.
+
 
 ```
-kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/v0.14.1/manifests.yaml
+autoKueue:
```

Comment:
is there a way to create tabs here to show the exact setup for different providers? See for example: https://gateway-api-inference-extension.sigs.k8s.io/guides/#deploy-the-inferencepool-and-endpoint-picker-extension